### PR TITLE
Added annotations from file

### PR DESCRIPTION
**Pull request by vmanilo**

### CODE REVIEW

File: remo/annotation_utils.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# ILSVRC2012_val_00000003.JPEG,N01751748, 10 20 30 40
+from remo.domain import Annotation
+
+
+def parse_plain_csv_object_detection(file_path):
+    """
+    Args
+        file_path: path to annotations
+
+    :return: annotations
```
Comment (drewlr) [inline remo/annotation_utils.py]: @vmanilo can you please add a docstring?
also, what's annotations?

File: remo/domain/dataset.py [hunk 2]
```diff
@@ -186,6 +190,21 @@ def add_annotations_by_csv(self, path_to_annotation_file, annotation_set_id):
         # fetch the dataset 
         self.fetch()
 
+    def add_annotations_from_file(self, annotation_set_id, file_path, parser_function):
+        """
+        Args
+            file_path: path to annotation file
+            parser_function: function which receives file_path and returns annotations
```
[the remark this answers is not in the document]
Comment (vmanilo) [reply remo/domain/dataset.py]: Added example

File: remo/annotation_utils.py (hunk 1 above)
Comment (vmanilo) [reply remo/annotation_utils.py]: added example

File: remo/sdk.py [hunk 3]
```diff
@@ -271,8 +274,45 @@ def create_annotation_set(self, annotation_task, dataset_id, name, classes):
     # def upload_annotations(self, dataset_id, path, annotation_task):
     #    return self.api.upload_file(dataset_id, path, annotation_task)
 
-    def add_annotation(self, dataset_id, annotation_set_id, image_id, cls, coordinates=None, object_id=None):
-        return self.api.add_annotation(dataset_id, annotation_set_id, image_id, cls, coordinates, object_id)
+    def add_annotation(self, annotation_set_id, image_id, cls, coordinates=None, object_id=None):
+        return self.api.add_annotation(annotation_set_id, image_id, cls, coordinates, object_id)
+
+    # TODO: rename it
+    def add_new_annotation(self, annotation_set_id, image_id, annotation):
```
[the remark this answers is not in the document]
Comment (vmanilo) [reply remo/sdk.py]: the idea was to remove existing `add_annotation` function, and replace it with new one

File: remo/sdk.py [hunk 4]
```diff
@@ -242,6 +242,9 @@ def get_annotations(self, annotation_set_id, annotation_format='json', export_co
                                           export_coordinates=export_coordinates, full_path=full_path)
         return result
 
+    def get_annotation_info(self, dataset_id, annotation_set_id, image_id):
+        return self.api.get_annotation_info(dataset_id, annotation_set_id, image_id)
```
[the remark this answers is not in the document]
Comment (vmanilo) [reply remo/sdk.py]: added

**Pull request merged**